x/stakeibc/keeper/reward_converter.go

```diff
@@ -67,7 +67,7 @@ func (k Keeper) TransferRewardTokensHostToTrade(ctx sdk.Context, amount sdkmath.
 	// Import and use pfm data structures instead of this JSON if we can determine consistent module version...
 	// This transfer channel id is a channel on the reward Zone for transfers to the trade zone
 	// (not to be confused with a transfer channel on Stride or the Host Zone)
-	memoJSON := fmt.Sprintf(`"forward": {"receiver": "%s","port": "%s","channel":"%s","timeout":"10m","retries": 2}`,
+	memoJSON := fmt.Sprintf(`"{ forward": {"receiver": "%s", "port": "%s", "channel":"%s", "timeout":"10m", "retries": 2} }`,
```

Comment:
Good catch, was this failing to parse before?

Reply:
I wasn't seeing an error for it, but I think it would have appeared on noble (and I doubt they have it logged by default)

I just noticed it visually when I was trying to recreate the transfer manually to confirm pfm was working in dockernet